manimlib/mobject/svg/svg_mobject.py

```diff
@@ -190,7 +194,10 @@ def get_mobject_from(self, shape):
         return None
 
     @staticmethod
-    def apply_style_to_mobject(mob, shape):
+    def apply_style_to_mobject(
+        mob: VMobject,
+        shape: se.Shape | se.Text
```

Comment:
`shape` can be simply annotated with `se.GraphicObject`. Similar for `get_mobject_from` method.